examples/django/requirements.txt

```
@@ -1,3 +1,3 @@
 django>=2.2.1
-sqlalchemy>=1.0.14
+sqlalchemy==1.4
```

We can relax this requirement can't we?
```suggestion
sqlalchemy>=1.4.0
```